11.0/apache/Dockerfile

```diff
@@ -95,7 +95,7 @@ RUN { \
 
 VOLUME /var/www/html
 
-RUN a2enmod rewrite
+RUN a2enmod rewrite remoteip ; { echo RemoteIPHeader X-Real-IP ; echo RemoteIPTrustedProxy 10.0.0.0/8 ; echo RemoteIPTrustedProxy 172.16.0.0/12 ; echo RemoteIPTrustedProxy 192.168.0.0/16 ; } > /etc/apache2/conf-enabled/remoteip.conf; chown www-data:root /etc/apache2/conf-enabled/remoteip.conf
```

Comment:
having the entire run statement in one line makes it difficult to read.
It would look a lot nicer if you formated it similar to the run statement above that sets the recommended PHP.ini settings